src/Tizen.Applications.Common/Tizen.Applications/ApplicationRunningContext.cs

```diff
@@ -255,6 +255,32 @@ public void Terminate()
             }
         }
 
+        /// <summary>
+        /// Terminates the background application.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when failed because of permission denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of system error.</exception>
+        /// <privilege>http://tizen.org/privilege/appmanager.kill</privilege>
```

Comment:
app_manager_request_terminate_bg_app() is a public API with http://tizen.org/privilege/appmanager.kill.bgapp privilege. I think this API can be public also.